website/cue/reference/components/sources/socket.cue

```diff
@@ -104,6 +104,22 @@ components: sources: socket: {
 				examples: ["/path/to/socket"]
 			}
 		}
+		socket_file_mode: {
+			common:        false
+			description: """
+				Unix file mode bits to be applied to the unix socket file
+				as its designated file permissions.
+				Note that the file mode value can be specified in any numeric format
+				supported by TOML, but it'd be more intuitive to use an octal number.
```

Comment:
```suggestion
				Note that the file mode value can be specified in any numeric format
				supported by your configuration language, but it is most intuitive to use an octal number.
```

We also support YAML (which does seem to support octal numbers) and JSON (which does not).